src/ORM/BehaviorRegistry.php

```diff
@@ -76,7 +76,11 @@ public function __construct(Table $table)
      */
     protected function _resolveClassName($class)
     {
-        return App::className($class, 'Model/Behavior', 'Behavior');
+        $result = App::className($class, 'Model/Behavior', 'Behavior');
+        if (!$result && in_array($class, ['CounterCache', 'Timestamp', 'Tree', 'Translate'])) {
+            return 'Cake\ORM\Behavior\\' . $class . 'Behavior';
```

Comment:
This is going to be a pain to maintain in the future. Wouldn't another call to App:classname() also work here?